server/bastionai_learning/src/optim/optimizer.rs

```diff
@@ -15,5 +28,10 @@ impl Optimizer for COptimizer {
     fn step(&mut self) -> Result<(), TchError> {
         COptimizer::step(self)
     }
+    fn into_bytes(&mut self) -> Result<Vec<u8>, TchError> {
```

Comment:
As we won't be able to get state from those, maybe we can just drop support for tch optimizers